src/Components/Attributes/Attributes.js

```diff
@@ -110,12 +110,16 @@ class Attributes extends React.Component {
     // `dataStructure.meta.id` is name of the data structure.
     const dataStructuresIndex = props.dataStructuresIndex || (
       reduce(dataStructures, (result, dataStructure) => {
-        result[dataStructure.meta.id] = dataStructure;
+        if (typeof dataStructure.meta.id === 'object') {
+          result[dataStructure.meta.id.content] = dataStructure;
+        } else {
+          result[dataStructure.meta.id] = dataStructure;
+        }
+
```

Comment:
I think this fix should be outside this PR (as the discussion here might take a bit)